specs/binding-filesystem.spec/src/main/scripts/io/aklivity/zilla/specs/binding/filesystem/streams/application/write.file.payload/client.rpt

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+connect "zilla://streams/app0"
+    option zilla:window 8192
+    option zilla:transmission "half-duplex"
+
+write zilla:begin.ext ${filesystem:beginEx()
+                                  .typeId(zilla:id("filesystem"))
+                                  .capabilities("WRITE_PAYLOAD")
+                                  .path("index.html")
+                                  .type("text/html")
+                                  .payloadSize(77)
+                                  .tag("f2c912a30f38a124c3249f8e802e0d90")
```

Comment:
What is the purpose of `tag` when writing the new payload?
Perhaps it is equivalent to `if-match` in `http` `PUT` ?